config.js

```diff
@@ -108,5 +108,6 @@ module.exports = {
       process.env[(process.platform == 'win32') ? 'USERPROFILE' : 'HOME'] + '/.barracks'
     )
   },
-  debug: process.env.DEBUG || false
-};
\ No newline at end of file
+  debug: process.env.DEBUG || false,
+  experimental: process.env.EXPERIMENTAL ? parseInt(process.env.EXPERIMENTAL) : false
```

Comment:
Rename to BARRACK_ENABLE_EXPERIMENTAL ?
Since it's an environment variable, just "experimental" seems a bit too generic to me